scripts/install2

```diff
@@ -11,6 +11,10 @@ CJDNS_BUILD_CMD="sudo Seccomp_NO=1 NO_NEON=1 ./do"
 if [[ $RPI_REVISION == *"900092"* || $RPI_REVISION == *"900093"* ]]; then
     echo -e "\e[1;32mStarting installation on Raspberry Pi Zero...\e[0m"
     CJDNS_BUILD_CMD="sudo Seccomp_NO=1 NO_NEON=1 CFLAGS=\"-s -static -Wall\" ./do"
+elif [[ `cat /proc/cpuinfo  | grep Hardware | awk '{print $3}'` == 'Allwinner' ]]; then
+    echo -e "\e[1;32mStarting installation on Orange Pi ???... \e[1;31m(EXPERIMENTAL) \e[0m"
+    CJDNS_BUILD_CMD="sudo Seccomp_NO=1 CFLAGS=\"-s -static -Wall -mfpu=neon -mcpu=cortex-a7 -mtune=cortex-a7 -fomit-frame-pointer -marm\" ./do"
+    RPI_REVISION="OPI"
```

Comment:
`OPI` is not a `RPI_REVISION`. You need to at the top of the file scan for whether this is a Raspberry Pi board or Orange Pi board and have a `BOARD_FAMILY` or similarly named variable, then go into revision detect login after matching per board family. It also needs stronger checks than just `Allwinner`.